libs/mintycore/src/main/java/com/solanamobile/mintyfresh/mintycore/endpoints/BundlrResponse.kt

```diff
@@ -0,0 +1,90 @@
+package com.solanamobile.mintyfresh.mintycore.endpoints
+
+import com.solanamobile.mintyfresh.mintycore.bundlr.DataItem
+import kotlinx.serialization.Serializable
+import kotlinx.serialization.SerializationException
+import kotlinx.serialization.json.*
+import okhttp3.RequestBody
+import okhttp3.RequestBody.Companion.toRequestBody
+import okhttp3.ResponseBody
+import retrofit2.Converter
+import retrofit2.Retrofit
+import java.lang.reflect.Type
+
+@Serializable
+data class NodeInfoResponse(
+    val version: String,
+    val gateway: String,
+    val addresses: NodeAddresses
+)
+
+@Serializable
+data class NodeAddresses(
+    // NOTE: there are other chains returned, but we only care about sol
+    val solana: String
+)
+
+@Serializable
+data class BalanceResponse(val balance: Long)
+
+@Serializable
+data class FundAccountResponse(val confirmed: Boolean)
+
+sealed interface UploadResponse {
+    val id: String
+
+    @Serializable
+    data class UploadComplete(override val id: String, val timestamp: Long) : UploadResponse
+    data class FileAlreadyUploaded(override val id: String) : UploadResponse
+}
+
+object TestConverter : Converter.Factory() {
```

Comment:
Is this only used for testing? Should probably be in that directory structure, not here.